src/config/constants/farms.ts

```diff
@@ -38,6 +38,17 @@ const farms: FarmConfig[] = [
   /**
    * V3 by order of release (some may be out of PID order due to multiplier boost)
    */
+  {
+    pid: 449,
+    lpSymbol: 'BP-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x2bF2dEB40639201C9A94c9e33b4852D9AEa5fd2D',
+    },
+    token: tokens.bp,
+    quoteToken: tokens.wbnb,
+    isCommunity: true,
```

Comment:
Should this one be labeled as Community?

Reply:
It shouldn't! Thanks for catching that.